src/components/Modals/ContactSelectorModal/ContactSelectorModalContent.js

```diff
@@ -181,6 +181,7 @@ const ContactSelectorModalContent = ({ chain, contacts = [], onSelectContact, qu
         onQueryChange={onQueryChange}
         placeholder={tRoot('label.addressSupportedNames')}
         error={!!errorMessage}
+        inputStyle={{ height: 78 }}
```

Comment:
This might be okay on your screen but on smaller screens this may be an issue. Could we dynamically determine the height of the box and use that value?

Reply:
I checked the smallest device which is not facing any issues.
![image](https://user-images.githubusercontent.com/78720269/216087960-5abd379d-dfa2-480a-9711-4c274b7067a7.png)